mmdet/models/seg_heads/panoptic_fusion_heads/heuristic_pan_head.py

```diff
@@ -0,0 +1,91 @@
+import torch
+
+from mmdet.datasets.coco_panoptic import INSTANCE_OFFSET
```

Comment:
Considering pass it INSTANCE_OFFSET through img_meta as it might change if we change a dataset

Reply:
This number works on the datasets with fewer than 1,000 categories.